src/internal/operators/switchMap.ts

```diff
@@ -56,7 +56,7 @@ export function switchMap<T, R, O extends ObservableInput<any>>(
  * // ... and so on
  * ```
  *
- * Rerun an interval Observable on every click event
+ * Return an interval Observable on every click event
```

Comment:
This probably isn't a typo and the author likely intended to use 'rerun', but your opening of this PR kinda shows that it's not as clear as it could be. Let's use 'restart' instead:
```suggestion
 * Restart an interval Observable on every click event
```